src/components/Square.tsx

```diff
@@ -0,0 +1,12 @@
+interface Props {
+  value: string;
+  onClick(): void;
+}
+function Square(props: Props) {
```

Comment:
Option: for `Props`, usually prefer `function Square({ value, onClick }: Props)`